guides/doc-Configuring_User_Authentication/master.adoc

```diff
@@ -0,0 +1,65 @@
+include::common/attributes.adoc[]
+include::common/header.adoc[]
+:context: authentication
+
+= {ConfiguringUserAuthenticationDocTitle}
+
+// This guide is not ready for stable releases
+ifdef::HideDocumentOnStable[]
+include::common/modules/snip_guide-not-ready.adoc[]
+endif::[]
+ifndef::HideDocumentOnStable[]
+
+ifdef::satellite[]
+include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[leveloffset=+1]
+endif::[]
+
+include::common/assembly_accessing-server.adoc[leveloffset=+1]
+
+include::common/assembly_configuring-an-ldap-server-as-an-external-identity-provider-for-project.adoc[leveloffset=+1]
+
+include::common/assembly_configuring-kerberos-single-sign-on-with-freeipa-in-project.adoc[leveloffset=+1]
+
+// The following ifdef sets :keycloak: to `Red Hat Single Sign-On` for Satellite builds only. For, non-Satellite builds, :keycloak: stays set to `Keycloak`.
+ifdef::satellite[]
+:keycloak: {keycloak-wildfly}
+:keycloak-example-com: rhsso.example.com
+endif::[]
+ifdef::context[:parent-context: {context}]
+:context: keycloak-wildfly
+include::common/assembly_configuring-keycloak-wildfly-authentication-for-project.adoc[leveloffset=+1]
+ifdef::parent-context[:context: {parent-context}]
+ifndef::parent-context[:!context:]
+// The following ifdef unsets :keycloak: because in Satellite builds, it's always either {keycloak-wildfly} or {keycloak-quarkus}.
+ifdef::satellite[]
+:!keycloak:
+endif::[]
+
+ifndef::satellite[]
+// The following ifdef sets :keycloak: to `Red Hat build of Keycloak` for Satellite builds only. For, non-Satellite builds, :keycloak: stays set to `Keycloak`.
+ifdef::satellite[]
```

Comment:
Is that really necessary if there's line 38?

Reply:
I added that because there is a good chance we will want to include the content for Satellite builds. In any case, it's all just temporary because the Wildfly distribution is going to go EOL soon. I'd rather not touch this mess again -- it works now and will be dropped soon :) No change.